docs/containers.conf.5.md

```diff
@@ -227,6 +227,13 @@ Path to the container-init binary, which forwards signals and reaps processes
 within containers. Note that the container-init binary will only be used when
 the `--init` for podman-create and podman-run is set.
 
+**interface_name**=""
+
+Default way to set interface names inside containers. Defaults to legacy pattern
+of ethX, where X is an integer, when left undefined.
+Options are:
+  `host`   Set same as the host interface name of the respective bridge.
```

Comment:
I would say uses the network_interface name from the network config as interface name. Depending on the driver this can be anything (bridge name for bridge but parent name for macvlan/ipvlan) so it is not limited to bridge in any way

Reply:
Sounds good.

Reply:
Based on this feedback, renamed the property value to `device` as that is the term used in the `ip` command manual.